question_thirteen.py

```diff
@@ -0,0 +1,42 @@
+from functools import reduce
+
+class Number:
+    def __init__(self, numbers):
+        self.numbers = numbers
+    
+    def get(self):
+        return self.numbers
+    
+    def change_original_values(self, func: lambda i: i): 
+        new_numbers = []
+        new_numbers = list(map(lambda i: i*2, self.numbers))
+        return new_numbers
+    
+    def filter_values(self, filter_func: lambda x: x):
+
+        filtered_numbers = [] 
+        filtered_numbers = list(filter(lambda i: (i % 2 == 0), self.numbers))
+        return filtered_numbers
+
+    def compound_the_numbers(self, reduce_func: lambda compound, d: compound + d):
+        compounded_value = reduce((lambda x, y: x + y), self.numbers)
```

Comment:
Same here. What If I need to perform multiple of all instead of summation? Compound means, you just need to provide one function that will be processed on the list and give you one single value in the output.